### ISSUE TITLE
Meta::ddl does not correctly export view contents from generated code

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#10594]
Generate code for any view like this:

```sql
CREATE VIEW v (a) AS SELECT 1;
```

The generated code might contain something like:

```sql
TableOptions.view("CREATE FORCE VIEW ...")
```

But nothing is generated when exporting the view using `Meta::ddl`. Instead, we're getting an empty definition like:

```sql
create view v(a)
as
;
```

**Comment by lukaseder** [comment]
In order to solve this, we'll have to parse the view and re-construct it in order to be sure all of our `Settings` (e.g. schema mapping) are applied correctly

**Comment by lukaseder** [comment]
There have been substantial investments in various involved areas (DDL support, parser, etc.). I don't have high hopes of being able to reasonably backport this fix. This will be fixed in jOOQ 3.14 only

**Comment by Rosenrot48** [comment]
Hi @lukaseder!

Has that bug been fixed? I use jooq version 3.17 and still have the same issue. 


JDBC driver name and version
org.postgresql:postgresql:42.3.8

> There have been substantial investments in various involved areas (DDL support, parser, etc.). I don't have high hopes of being able to reasonably backport this fix. This will be fixed in jOOQ 3.14 only

**Comment by lukaseder** [comment]
@Rosenrot48 Yes, it has been fixed. It is likely you donot have the exact same issue. Please open up a new issue with all the usual details.
